nav2_docking/opennav_docking/test/test_docking_server.py

```diff
@@ -96,10 +109,14 @@ def tearDownClass(cls) -> None:
 
     def setUp(self) -> None:
         # Create a ROS node for tests
+        self.period = 0.15  # 30Hz
```

Comment:
The comment here doesn't make sense? If period = 0.15, then that's ~6.6hz, not 30hz? Also, how does this work correctly when the timer below is then setup to run at 30hz, but the period doesn't match that?